packages/ses/src/lockdown-shim.js

```diff
@@ -228,8 +206,9 @@ export const repairIntrinsics = (options = {}) => {
 
   if (seemsToBeLockedDown()) {
     // eslint-disable-next-line @endo/no-polymorphic-call
-    console.log('Seems to already be locked down. Skipping second lockdown');
-    return alreadyHardenedIntrinsics;
+    throw new TypeError(
+      `Already locked down but not by this SES instance (SES_MULTIPLE_INSTANCES)`,
+    );
   }
```

Comment:
This makes clear which of the two conditions we treat as an error: where both lockdowns are called. This seems sensible.